docs/source/conf.py

```diff
@@ -1,12 +1,13 @@
 # -*- coding: utf-8 -*-
 #
-import sys
 import os
 import shlex
+import sys
 
-# For conversion from markdown to html
 import recommonmark.parser
 
+# For conversion from markdown to html
```

Comment:
Let's remove this since it was for the import in line 7.

Reply:
Yeah, looks like a few import-related comments got moved to weird places. Will have to check those